code/src/components/Header.js

```diff
@@ -0,0 +1,146 @@
+import React, {useState, useEffect} from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import Button from "styles/Button";
+import { useNavigate } from "react-router-dom";
+
+
+
+
+const Heading = styled.header `
+  background: #000;
+  display: flex;
+  justify-content: flex-end;
+  align-item: center;
+  padding: 10px;
+
+`
+
+const SortMenu = styled.select`
+  width: 150px;
+  height: 50px;
+  background: #000;
+  color: #fff;
+  border: transparent;
+  outline: none;
+  font-size: 18px;
+  font-weight: bold;
+  appearance: none;
+  padding: 10px;
+  text-align: center;
+  cursor: pointer;
+
+
+  &:focus {
+    border: solid 2px red;
+}
+
+option {
+  color: black;
+  background: white;
+  display: flex;
+  white-space: pre;
+  min-height: 20px;
+  padding: 0px 2px 1px;
+
+}
+`
+// const TitleText = styled.h1`
```

Comment:
This code is commented out. Are you saving it for later use? Otherwise perhaps it could be removed?